lib/systems/examples.nix

```diff
@@ -90,6 +90,11 @@ rec {
     config = "mipsel-unknown-linux-gnu";
   } // platforms.fuloong2f_n32;
 
+  loongnix = {
+    config = "loongarch64-unknown-linux-gnu";
+    gcc = { arch = "loongarch64"; };
```

Comment:
What's this line required for?

Reply:
Can you explain it in detail? I not sure what did you mean.